test/runtests.jl

```diff
@@ -164,178 +164,178 @@ end
         convTol=1e-8)
     test_show(gm9)
     @test dof(gm9) == 3
-    @test isapprox(deviance(gm9), 0.16260829451739)
-    @test isapprox(loglikelihood(gm9), -26.24082810384911)
-    @test isapprox(aic(gm9), 58.48165620769822)
-    @test isapprox(aicc(gm9), 63.28165620769822)
-    @test isapprox(bic(gm9), 59.07332993970688)
-    @test isapprox(coef(gm9), [5.50322528458221, -0.60191617825971])
-    @test isapprox(GLM.dispersion(gm9.model, true), 0.02435442293561081)
-    @test isapprox(stderr(gm9), [0.19030107482720, 0.05530784660144])
+@test isapprox(deviance(gm9), 0.16260829451739)
```

Comment:
I am baffled. I went back to a version of the file before this comment and pasted in what it had above my additions, verbatim. If this didn't work, I don't know what will.